### ISSUE TITLE
Suggestion:  Save plan as .mavlink

### ISSUE THREAD

**Issue by mgr098** [mavlink/qgroundcontrol#10342]
I want to use QGroundControl Plan view to create plans for my ANAFI AI drone which supports MAVlink. However, you can only save plans as a .plan file, while the drone only allows .mavlink format. Since QGC in effect converts the plan to a mavlink format it would be nice if you could download the mavlink file directly. This would allow other drone developers to use QGC plans for drones that do not support .plan types.

**Comment by hamishwillee** [comment]
@mgr098 Yes, it would be convenient if QGC could export a plan in ["plain text format"](https://mavlink.io/en/file_formats/#mission_plain_text_file). I've moved it here to the qgc repo, since this not a mavlink question.

What would be even better would be a discussion around:
- standardization of the plan format
- clear definition of the algorithms used to map survey to mavlink commands so these could be reproduced and round-tripped. This might simply be survey markers grouping survey items that a ground station could interpret.
